pkgs/applications/emulators/emulationstation-de/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, stdenv, fetchFromGitLab, cmake, SDL2, curl, ffmpeg, pugixml, pkgconf, freetype, freeimage, alsa-lib }:
+
+stdenv.mkDerivation rec{
+  pname = "emulationstation-de";
+  version = "1.2.6";
+
+  src = fetchFromGitLab {
+    owner  = "es-de";
+    repo   = "emulationstation-de";
+    rev    = "v${version}";
+    sha256 = "BiQnHtcKheEhwp0KKy9BCDIuZuAjmS8tWNyNw4nl5Fk=";
+  };
+
+  patches = [ ./es-find-rules.patch ];
```

Comment:
Rename the patch to a more helpful filename, like `001-add-nixpkgs-cores-directory.diff`.